docs/libcurl/opts/CURLOPT_FOLLOWLOCATION.md

```diff
@@ -64,6 +68,39 @@ client may not want to pass on to other servers than the initially intended
 host and for all other headers than the two mentioned above, there is no
 protection from this happening when libcurl is told to follow redirects.
 
+Pick one of the following modes:
+
+## CURLFOLLOW_ALL (1)
+
+Before 8.13.0 this bit had no name and 1L was just the value to enable this
+option. This makes a set custom method be used in all HTTP requests, even
+after redirects.
+
+## CURLFOLLOW_OBEYCODE (2)
+
+When there is a custom request method set with CURLOPT_CUSTOMREQUEST(3), that
+set method replaces what libcurl would otherwise use. If a 301/302/303
+response code is return to signal a redirect, the method is changed from POST
+to `GET`. For 307/308, the custom method remains set and used.
+
+Note that only POST (or a custom post) is changed to GET on 301/302. Not PUT
+etc - and not when libcurl issues a custom PUT. 303 makes it switch to GET
+independently of the original method (except for HEAD).
+
+If you prefer a custom POST method to be reset to exactly the method `POST`,
+use CURLFOLLOW_FIRSTONLY instead.
+
+## CURLFOLLOW_FIRSTONLY (3)
+
+When there is a custom request method set with CURLOPT_CUSTOMREQUEST(3), that
+set method replaces what libcurl would otherwise use in the first outgoing
+request only.
```

Comment:
What happens *after* the first request? Is it `CURLFOLLOW_ALL` behavior or `CURLFOLLOW_OBEYCODE` behaviour or something else entirely? How about after 301/302/303? The next paragraphs imply that it could be `CURLFOLLOW_ALL` or CUSTOMREQUEST is disabled, but it should be made clear.

Reply:
I added some text. Maybe it needs even more.